### DISCUSSION TITLE
Types not identified correctly for $t in Typescript codebase

### DISCUSSION THREAD

**Discussion by aakash14goplani** [sveltekit-i18n/lib#129] [Q&A] [answered]
Hello @jarda-svoboda 

I am getting following error in VS-Code when using `$t`

<img width="655" alt="image" src="https://github.com/sveltekit-i18n/lib/assets/13452115/cbad6d18-7eb2-4d44-a0c3-a262cedc3ecc">   

   
**Corresponding plural string**
```json
"awards": "You have {{award:gt; 0:{{not won any awards}} {{award; 1:won exactly {{award}} award; default:won {{award}} awards}}; default:not won any awards}}!",
```

Although functionality works correctly and there are no errors thrown at build and compile time, it's just my VSCode that is annoying me. 

**Configuration:**

*package.json*
```json
"dependencies" : {
  "sveltekit-i18n": "latest"
}
```

*tsconfig*
```json
"compilerOptions": {
  "types": ["vite/client", "svelte", "sveltekit-i18n"]
}
```

Can you please let me know how to fix them, looking upto examples didn't help as they are not configured in TS!

**Comment by jarda-svoboda** [comment, marked as answer]
Hi @aakash14goplani!
You can add your custom types [this way](https://github.com/sveltekit-i18n/lib/issues/60#issuecomment-1320980999).
